.github/workflows/on_pull_request_package_size.yml

```diff
@@ -6,6 +6,9 @@ jobs:
   package-size:
     name: Check package sizes
     runs-on: ubuntu-latest
+    env:
```

Comment:
Had to add this in. It was previously using the token in the nx.json but now had to use env variable